conf/pki_provider.conf.example

```diff
@@ -0,0 +1,15 @@
+[entry1]
```

Comment:
This configuration file is not an example... these values and parameters don't mean anything and they're not even commented